process/economics/economicsData.go

```diff
@@ -348,6 +361,23 @@ func (ed *economicsData) getTotalFeesRequiredForInnerTxs(innerTxs []data.Transac
 	return totalFees
 }
 
+func (ed *economicsData) isMoveBalance(tx data.TransactionHandler) bool {
```

Comment:
this function is not good. 

you would also need to check on line 373 function to be not a built in function. and you can have execution of contracts without arguments. with only function call.

Reply:
we need the txTypeHandler here - or to have the txType added as input parameter in the function.

Reply:
I was afraid this won't be enough.. will think about another solution.. having txTypeHandler here won't work, as it is created later on, during process components creation

Reply:
added a setter for the real tx type handler and used it